daemon/sm.c

```diff
@@ -808,16 +808,14 @@ sm(
 }
 
 
-MTC_STATIC void
-rendezvous(
-    SM_PHASE    phase1,
-    SM_PHASE    phase2,
-    SM_PHASE    phase3,
-    MTC_BOOLEAN on_heartbeat,
-    MTC_BOOLEAN on_statefile)
-{
 #if RENDEZVOUS_FAULT_HANDLING
-    void rendezvous_wait(p1, p2)
+
+static void
```

Comment:
The entire rest of the project uses `MTC_STATIC` which expands to nothing.  https://github.com/xenserver/xha/blob/ae864a308337cc01395e9031c44f5dc4fbe3d480/include/mtctypes.h#L455-L461 says this is for backtrace reasons.

Anyway, you probably ought to stay consistent.

Reply:
Added back MTC_STATIC in the PR that is split from this